.github/workflows/snyk.yaml

```diff
@@ -0,0 +1,31 @@
+---
+name: Snyk Monitor
+on:
+  push:
+    branches:
+      - main
```

Comment:
@justinstoller ah this needs to be `master` or we need to change the default branch to `main`. If we do the latter then CJC needs to change (at least https://github.com/puppetlabs/ci-job-configs/blob/ae8e99bdb229b1cd28f3cdec845654bd8e681505/jenkii/enterprise/projects/ruby-hocon.yaml#L6 and maybe other places?)